docs/client/jdbc/kyuubi_jdbc.rst

```diff
@@ -110,14 +106,14 @@ Basic Connection URL format
 Use the connection URL to supply connection information to the kyuubi server or cluster that you are
 accessing. The following is the format of the connection URL for the Kyuubi Hive JDBC Driver
 
-.. code-block:: jdbc
+.. code-block::
 
-   jdbc:subprotocol://host:port/schema;<clientProperties;><[#|?]sessionProperties>
+   jdbc:kyuubi://host:port[/catalog]/[schema];<clientProperties;><[#|?]sessionProperties>
 
-- subprotocol: kyuubi or hive2
```

Comment:
Let‘s keep this, supporting `hive2` is an advantage, not something we need to cover up